buildtools/check-example

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import sys
+import tempfile
+
+
+def main():
```

Comment:
Would be helpful to have a short description of what does this script do and an example usage.
A idea about how to create a mask, some example using CLI ?

Reply:
Just done :-)